src/app/@core/services/local-storage.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LocalStorageService {
+	private storage: Storage;
+	private key: string = 'TokenLogin';
+	constructor(){
+		this.storage = window.localStorage;
+	}
+
+	public createToken(token: string): void{
+		if(this.storage){
+			this.storage.setItem(this.key, token);
```

Comment:
Colocar o JSON.stringify aqui no service, para não precisar chamar toda vez que for usar esse método.

Reply:
O Token já vem como string, não precisa usar o stringify